src/sourceMaps/sourceMap.ts

```diff
@@ -63,8 +63,9 @@ export class SourceMap {
 
         // Rewrite sm.sources to same as this._sources but forward slashes and file url
         sm.sources = this._sources.map(sourceAbsPath => {
-            // Convert to file: url. After this, it's a file URL for an absolute path to a file on disk with forward slashes.
-            return utils.pathToFileURL(sourceAbsPath);
+            // Convert to file:/// url. After this, it's a file URL for an absolute path to a file on disk with forward slashes.
+            // We lowercase so authored <-> generated mapping is not case sensitive
+            return utils.pathToFileURL(sourceAbsPath).toLowerCase();
```

Comment:
This should match `this._sources`, but if you're lowercasing it on the sourceMaps.ts end, it probably isn't necessary.